sdk/core/azure-core/pom.xml

```diff
@@ -83,6 +83,8 @@
     <compiler.failondeprecatedstatus/>
     <javadoc.excludePackageNames>com.azure.json</javadoc.excludePackageNames>
 
+    <spotbugs.skip>false</spotbugs.skip>
```

Comment:
is it false by default? Do we need to explicitly set this now?

Reply:
For now, this does need to be explicitly set. A few weeks ago there was a rush upgrade due to some transitive dependencies the old version of Spotbugs we used had. The newer version of Spotbugs added new checks we didn't have time to handle (and were under existing error codes) so it was disabled.